account_payment_show_invoice/README.rst

```diff
@@ -0,0 +1,84 @@
+============================
+Account Payment Show Invoice
+============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--payment-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-payment/tree/12.0/account_payment_show_invoice
+    :alt: OCA/account-payment
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-payment-12-0/account-payment-12-0-account_payment_show_invoice
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/96/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module extends the tree view of payments to show the paid invoices
+related to the payments using the vendor reference by default.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to 'Invoicing > Vendors > Payments' or to 'Invoicing > Customers >
+   Payments'
```

Comment:
I did a function test and did not found the menus mentioned here. Can you please check this too?

Reply:
![image](https://user-images.githubusercontent.com/1973598/55896541-b80e5c80-5be8-11e9-93d1-138e36e61361.png)